library/ctr_drbg.c

```diff
@@ -51,6 +51,7 @@
 void mbedtls_ctr_drbg_init( mbedtls_ctr_drbg_context *ctx )
 {
```

Comment:
Please add a changelog entry (see https://github.com/Kxuan/mbedtls/blob/fix-ctr_drbg-uninit/ChangeLog.d/00README.md) with a short explanation of what this fixes.